doc-layout/src/template.js

```diff
@@ -1,34 +1,42 @@
 import '@divriots/dockit-core/layout/dockit-layout.define.js';
 import { styles } from '@divriots/dockit-core/layout';
+import { setupSpeedyLinks } from '@divriots/dockit-core/speedy-links';
 import { html } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import logoSvg from './logo.svg?raw';
 
-export const docLayoutTemplate = (content, context) => html`
-  <style>
-    ${unsafeHTML(styles)} .logo {
-      color: #f8c307;
-    }
-    .preview-story .story_padded {
-      background-color: #f3f4f6;
-    }
-    html.dark .preview-story .story_padded {
-      background-color: #1f2937;
-    }
-  </style>
-  <dockit-layout
-    .context="${context}"
-    @color-scheme-change="${(event) => {
-      if (event.detail.colorScheme === 'dark') {
-        document.documentElement.classList.add('dark');
-      } else {
-        document.documentElement.classList.remove('dark');
+export const docLayoutTemplate = (content, context) => {
+  setupSpeedyLinks({
```

Comment:
would have been nice to just pass the context to the speedy link setup. I think if both have the same method api (same method name) it should work

Reply:
speedyLinks is generic, not coupled to the context object, therefore the interface is not the same and shouldn't be by design

Reply:
Context is just a container object ... making it with traits and passing directly the container will let more extensibility for the future instead of just this method, just my 2 cents

Reply:
pair programming might help, this is a bit far fetched for me, I don't understand the vision

Reply:
I am free tomorrow if you need